### PR TITLE
refactor: 💡 textarea focus and resizing

### PR DESCRIPTION
**Pull request by koepferd**
## Proposed Changes

  - fix: click on placeholder in textarea focusses the editor and is ready to type
  - refactor resizing logic

### CODE REVIEW

File: src/components/messageSubmitInterface/messageSubmitInterface.styles.scss [hunk 1]
```diff
@@ -217,22 +205,25 @@ $contentWidth: calc((100vw - #{$navWidth}) / 12 * 8);
 		}
 
 		&__toolbar {
-			width: calc(100% - 40px);
+			width: calc(100% - #{1px});
```
Comment (web-mi) [inline src/components/messageSubmitInterface/messageSubmitInterface.styles.scss]:
```suggestion
			width: calc(100% - 1px);
```

File: src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx [hunk 2]
```diff
@@ -862,29 +812,27 @@ export const MessageSubmitInterfaceComponent = (
...
-								type="file"
-								id="dataUpload"
-								name="dataUpload"
-								accept="image/jpeg, image/png, .pdf, .docx, application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
-							/>
+									</div>
+								))}
 						</span>
+						<SendMessageButton
+							handleSendButton={(event) =>
```
Comment (web-mi) [inline src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx]:
```suggestion
							handleSendButton={handleButtonClick}
```

File: src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx [hunk 3]
```diff
@@ -862,29 +812,27 @@ export const MessageSubmitInterfaceComponent = (
...
+								handleButtonClick(event)
```
Comment (web-mi) [inline src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx]:
```suggestion
```

File: src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx [hunk 4]
```diff
@@ -862,29 +812,27 @@ export const MessageSubmitInterfaceComponent = (
...
+							}
```
Comment (web-mi) [inline src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx]:
```suggestion
```

File: src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx [hunk 5]
```diff
@@ -361,80 +361,53 @@ export const MessageSubmitInterfaceComponent = (
...
-
-		resetTextareaSize(textarea);
-
-		let maxHeight;
+		const textInput: any = textareaInputRef.current;
+		// default values
+		let textareaMaxHeight;
 		if (window.innerWidth <= 900) {
-			maxHeight = 118;
+			textareaMaxHeight = 118;
```
Comment (web-mi) [inline src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx:368]: We schould consider if it makes sense to use styled-components in this case

**Review by web-mi — CHANGES_REQUESTED** [review]

File: src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx (hunk 5 above)
Comment (koepferd) [reply src/components/messageSubmitInterface/messageSubmitInterfaceComponent.tsx:368]: As discussed, generally yes, not in the scope of the bugfix more me though.
Let's take this to the story about general style improvements :-)

**Review by web-mi — APPROVED** [review]

**Pull request merged**